ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,214 @@
+                                "is sit amet elit nunc. In in ante dapibus, lobortis justo imperdiet, ultrices libero. Cras venen" +
+                                "atis eros id pulvinar venenatis. Mauris feugiat augue a sem dapibus vestibulum. Nam molestie nib" +
+                                "h convallis tincidunt consequat. Phasellus at ligula risus. Sed id nibh condimentum, pulvinar es" +
+                                "t vestibulum, semper magna. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque pla" +
+                                "cerat sem ut malesuada suscipit. Fusce massa tortor, tristique nec quam eget, volutpat sollicitu" +
+                                "din odio. Maecenas id iaculis elit. Vestibulum a neque aliquam, pulvinar felis quis, venenatis q" +
+                                "uam. In pretium sagittis auctor. Sed cursus sollicitudin neque, a mattis turpis lacinia mattis. " +
+                                "Phasellus varius sit amet sem non dapibus. Nullam dapibus orci ex, sed cursus augue auctor vitae" +
+                                ". Proin ac sagittis nulla. Morbi accumsan nisl augue, quis sodales arcu suscipit vitae. Sed quis" +
+                                " efficitur odio, ac sagittis odio. Etiam at dolor at justo ultrices gravida a nec risus. Nam tri" +
+                                "stique, risus ac condimentum suscipit, tellus lorem luctus enim, at aliquet metus magna sed urna" +
+                                ". Fusce id fringilla sapien. Donec congue tellus diam. Praesent et fermentum metus. In sit amet " +
+                                "lacus a ante consequat tincidunt. Sed pretium justo ut malesuada ultrices. Quisque sagittis port" +
+                                "titor tempus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia c" +
+                                "urae; Donec a consectetur mauris. In tempor volutpat risus, et aliquam elit tristique ut.Duis so" +
+                                "llicitudin lorem eget tincidunt tincidunt. Lorem ipsum dolor sit amet, consectetur adipiscing el" +
+                                "it. Proin ante tellus, tincidunt eu nunc a, efficitur suscipit metus. Donec nec sapien at massa " +
+                                "maximus lobortis a vitae diam. Cras maximus condimentum est ac mollis. Phasellus quis porttitor " +
+                                "erat. Curabitur dignissim, orci quis pulvinar suscipit, purus nunc hendrerit lacus, sed ultrices" +
+                                " leo eros ac est. Nulla at risus nisl. Pellentesque varius orci non erat aliquet, vitae facilisi" +
+                                "s leo porta. Fusce id mauris.";
+
+            char[] punctuations = { ' ', ',', '.', ';', ':' };
+            string[] LI_arr = loremipsum.Split(punctuations, StringSplitOptions.RemoveEmptyEntries);
+
+            int lorem_cnt = 0;
```

Comment:
use camelCase for naming variables 
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions